tests/cypress/support/utilities/utilities.js

```diff
@@ -29,7 +29,7 @@ class utilities {
     let num1 = Math.floor(Math.random() * Math.floor(80)) + 10;
     let num2 = Math.floor(Math.random() * Math.floor(80)) + 10;
     let num3 = Math.floor(Math.random());
-    // SS.#####.R00.00
+    // SS-#####.R00.00
     let waiverNumber = state + "." + num1 + "" + num2 + "" + num3 + ".R00.00";
```

Comment:
Did you mean to only change the comment and not the waiverNumber variable value?

Reply:
Oh I actually didn't mean to change that bit at all. We don't use that utility in the code anymore. At least, not right now.